storage/gcloud/aio/storage/storage.py

```diff
@@ -241,24 +241,26 @@ def _split_content_type(content_type: str) -> Tuple[str, str]:
 
     async def _upload_resumable(self, url: str, object_name: str,
                                 stream: io.IOBase, headers: dict, *,
+                                metadata: dict = None,
                                 session: aiohttp.ClientSession = None,
                                 timeout: int = 30) -> dict:
         # https://cloud.google.com/storage/docs/json_api/v1/how-tos/resumable-upload
         session_uri = await self._initiate_upload(url, object_name, headers,
-                                                  session=session)
+                                                  metadata, session=session, )
         data: dict = await self._do_upload(session_uri, stream,
                                            headers=headers, session=session,
                                            timeout=timeout)
         return data
 
     async def _initiate_upload(self, url: str, object_name: str, headers: dict,
-                               *,
+                               metadata: dict, *,
```

Comment:
Should this also be a kwarg with default `= None`?

Reply:
Previously I wanted to be just arg, but pylint was telling me that this function has too much arguments, so I changed it because of this